filesystem/src/main/java/com/nytimes/android/external/fs/SourcePersisterFactory.java

```diff
@@ -0,0 +1,28 @@
+package com.nytimes.android.external.fs;
+
+import com.nytimes.android.external.fs.filesystem.FileSystemFactory;
+import com.nytimes.android.external.store.base.Persister;
+
+import java.io.File;
+import java.io.IOException;
+
+import okio.BufferedSource;
+
+/**
+ * Created by Scott on 1/8/2017.
```

Comment:
did mean to keep the "created" thing, just IDE setting I forgot to change, I'll fix.